File: blogeur_ia_python/blogueur_ia/categories.py
from blogueur_ia.config import settings

CATEGORIES_BLOG = settings.categories
CATEGORIES_BLOG_IDS = {c["id"] for c in CATEGORIES_BLOG}
CATEGORIE_PRODUIT_ID = settings.categorie_produit_id
CATEGORIE_DEFAUT = settings.categorie_defaut

File: blogeur_ia_node/src/categories.ts
import { settings } from "./config";

export const CATEGORIES_BLOG = settings.categories;
export const CATEGORIES_BLOG_IDS = new Set(CATEGORIES_BLOG.map((c) => c.id));
export const CATEGORIE_PRODUIT_ID = settings.categorieProduitId;
export const CATEGORIE_DEFAUT = settings.categorieDefaut;